src/js/Maze.js

```diff
@@ -51,6 +51,42 @@ class Maze {
             return index ===cells.length ? false : true;
         }
     }
+
+    /**
+     * Calculate distance of all maze cells from cell at given index,
+     * then use color to visualize distance.
+     *
+     * @param {number} cellIdx - Index of selected start cell.
+     */
+    colorByDistanceFromCell(cellIdx) {
+        const cells = this.getCells();
+        cells.forEach((cell) => cell.distance = null);
```

Comment:
I still think I have a slight preference for tracking this in a dictionary as opposed to on the cell object - I don't really like putting extra properties on something that are incidental to actual existence. Then you could also skip initializing them as null and just have 
```
const distancesFromCell = {}
```